pinot-common/src/test/java/org/apache/pinot/common/utils/RoundRobinURIProviderTest.java

```diff
@@ -50,128 +52,135 @@ public void testHostAddressRoundRobin()
     mock.when(() -> InetAddress.getAllByName("testweb.com")).thenReturn(testWebAddresses);
 
     TestCase[] testCases = new TestCase[]{
-        new TestCase("http://127.0.0.1", new String[]{"http://127.0.0.1"}),
-        new TestCase("http://127.0.0.1/", new String[]{"http://127.0.0.1/"}),
-        new TestCase("http://127.0.0.1/?", new String[]{"http://127.0.0.1/?"}),
-        new TestCase("http://127.0.0.1/?it=5", new String[]{"http://127.0.0.1/?it=5"}),
-        new TestCase("http://127.0.0.1/me/out?it=5", new String[]{"http://127.0.0.1/me/out?it=5"}),
-        new TestCase("http://127.0.0.1:20000", new String[]{"http://127.0.0.1:20000"}),
-        new TestCase("http://127.0.0.1:20000/", new String[]{"http://127.0.0.1:20000/"}),
-        new TestCase("http://127.0.0.1:20000/?", new String[]{"http://127.0.0.1:20000/?"}),
-        new TestCase("http://127.0.0.1:20000/?it=5", new String[]{"http://127.0.0.1:20000/?it=5"}),
-        new TestCase("http://127.0.0.1:20000/me/out?it=5", new String[]{"http://127.0.0.1:20000/me/out?it=5"}),
+        new TestCase("http://127.0.0.1", Arrays.asList("http://127.0.0.1")),
```

Comment:
Why changing this? I do get an IDE warning when using `Arrays.asList` on single element

Reply:
The purpose is to leverage some APIs from `List` interface like indexOf(), which doesn't exist in array. I can update those which have IDE warning to `Collections.singletonList()` in the next push.